mailing_list/management/commands/activate_all_lists.py

```diff
@@ -19,21 +19,26 @@ class Command(BaseCommand):
     help = 'Activates all mailing lists for a given account and enrollment term'
 
     def add_arguments(self, parser):
-        parser.add_argument('--account-id', type=int, required=True,
-                            help='Canvas account id')
-        parser.add_argument('--enrollment-term-id', type=int, required=True,
-                            help='Canvas enrollment term id')
+        parser.add_argument('--account-id', required=True,
+                            help='Canvas account id (e.g. 29) or SIS account ID (e.g. sis_account_id:school:hls)')
+        parser.add_argument('--enrollment-term-id', nargs='*', required=True,
```

Comment:
Just curious...does `nargs='*', required=True` require the option, but allow for no values to it?  ie. lets them explicitly not set it, but won't let you forget to pass it?